codespell_lib/data/dictionary.txt

```diff
@@ -1655,6 +1657,7 @@ ba->by, be,
 bacause->because
 baceause->because
 bacic->basic
+back ground->background
```

Comment:
Codespell can't currently handle this. Please remove it and add it to the list in https://github.com/codespell-project/codespell/issues/74 .

Reply:
done